### ISSUE TITLE
Built-in modules of edgeworkers

### ISSUE THREAD

**Issue by Krishjain** [AustinGil/edgeworkers-local-server#1]
Any chance you can add the rest of the built-in modules of edgeworkers like streams and text-encode-transform?

**Comment by AustinGil** [comment]
Yeah, I'm planning to continue building this out, but am currently a bit under pressure with some other work. Things should open up next month though.
